test/fixtures/restaurants.yml

```diff
@@ -4,7 +4,7 @@ one:
   name: MyString
```

Comment:
Are you using this fixture? Delete it if not.

(Fixture gives you default data to work with in testing)

Reply:
The same, I have used FactoryBot for fixtures, I removed those default fixtures files
Thanks